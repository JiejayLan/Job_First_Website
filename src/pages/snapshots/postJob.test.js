import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import PostJob from "../postJob"

describe("postJob snapshot test", () => {
    test('postJob render should match snapshot', () => {
        const render = new ReactShallowRenderer();
        render.render(<PostJob />);
        expect(render.getRenderOutput()).toMatchSnapshot();
    });
});