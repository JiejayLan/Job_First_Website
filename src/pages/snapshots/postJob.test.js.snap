// Snapshot of correct postJob page